### ISSUE TITLE
NTR: Renal biopsy: collagen alpha3(IV) chain absent in renal basement membranes by immunohistochemical analysis (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9126]
New term request Renal biopsy: collagen alpha3(IV) chain absent in renal basement membranes by immunohistochemical analysis (elthat_100513012247): Kidney

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6001027
- label: Diminished glomerular basement membrane collagen IV alpha 3 chain staining
- parents: [Abnormal glomerular basement membrane morphology (HP:0033282)](https://hpo.jax.org/app/browse/term/HP:0033282)
- pmid: PMID:8793804
- def: Abnormal reduction in the amount of the alpha 3 chain of collagen IV in the basement membrane of the glomeruli of the kidney.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9126
- synonyms:
